client/device/asa/create.go

```diff
@@ -206,7 +218,18 @@ func Create(ctx context.Context, client http.Client, createInp CreateInput) (*Cr
 	// poll until asa config state done
 	client.Logger.Println("waiting for device to reach state done")
 
-	err = retry.Do(ctx, asaconfig.UntilStateDone(ctx, client, asaReadSpecOutp.SpecificUid), *retry.NewOptionsWithLogger(client.Logger))
+	err = retry.Do(
+		ctx,
+		asaconfig.UntilStateDone(ctx, client, asaReadSpecOutp.SpecificUid),
+		retry.NewOptionsBuilder().
+			Message("waiting for ASA configuration update to be done").
```

Comment:
Waiting for ASA to be onboarded to CDO...

Reply:
done 👍